starknet_devnet/chargeable_account.py

```diff
@@ -0,0 +1,24 @@
+"""
+Account that is charged with a fee when nobody else can be charged.
+"""
+
+from starknet_devnet.account import Account
+
+
+class ChargeableAccount(Account):
+    """
+    A well-funded account that can be charged with a fee when no other account can.
```

Comment:
More comments about why we need it and how it can be useful in future.

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/384/commits/4de387aeb30c36f3feea8eae0b4e9d1d3a571852